/**
 * Copyright (c) Contributors as noted in the AUTHORS file.
 *
 * This Source Code Form is part of *fuurin* library.
 *
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */

syntax = "proto3";

import "google/protobuf/empty.proto";

message SeqNum {
    uint64 value = 1;
}

service WorkerService {
    rpc GetSeqNum(google.protobuf.Empty) returns (SeqNum) {}
}
